uint8 isObj
uint8 numObj
uint8[] IdObj
uint8[] isApproach
float32[] alpha
float32[] posX
float32[] posY
float32[] distance
float32[] velocity
float32[] ttc
